go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/multiversx/mx-chain-crypto-go v1.2.8
 	github.com/multiversx/mx-chain-logger-go v1.0.13
 	github.com/multiversx/mx-chain-scenario-go v1.2.1
-	github.com/multiversx/mx-chain-vm-common-go v1.5.5
+	github.com/multiversx/mx-chain-vm-common-go v1.5.6-0.20231025104250-d24edf1e19ae
```

Comment:
let's not forget about the proper release

Reply:
added